src/workerd/api/global-scope.c++

```diff
@@ -3,6 +3,7 @@
 //     https://opensource.org/licenses/Apache-2.0
 
 #include "global-scope.h"
+#include "workerd/jsg/promise.h"
```

Comment:
style nit: this should be `#include <workerd/jsg/promise.h>` and should be grouped with the other imports from `workerd/jsg/...` further down.